build/docker/kmesh.dockerfile

```diff
@@ -7,11 +7,13 @@ WORKDIR /kmesh
 
 ARG arch
 
+RUN \
+    --mount=type=cache,target=/var/cache/dnf \
+    yum install -y kmod util-linux iptables
```

Comment:
I tried this, and got a error:
Error response from daemon: Dockerfile parse error line 13: Unknown flag: mount

Is there any requirement on the Docker version?

Reply:
[error-response-from-daemon-dockerfile-parse-error-unknown-flag-mount](https://stackoverflow.com/questions/55153089/error-response-from-daemon-dockerfile-parse-error-unknown-flag-mount) 
May be Docker version is too old？Or trying add a comment `# syntax=docker/dockerfile:experimental` in first line of Dockerfile as above link said.